### ISSUE TITLE
Open source forum

### ISSUE THREAD

**Issue by julianfairfax** [ProtonMail/WebClients#260]
I would post this on the forum, as it's a feature request, but given the nature of the request, I don't feel that would be appropriate. Your forum is just simply not in the spirit of your brand. You're using a closed source third-party provider to host an outdated and visually unappealing forum. An open source solution such as https://www.discourse.org/ can be fully self-hosted by you with minimal resources being used on what I'm sure are already a good number of servers you have. It's a free, open source, modern, and well-designed system. If an open source forum was used, it would turn from an unappreciated corner of your brand that no one wants to use, to a well-implemented space for the community's ideas. Please consider adopting the use of a platform such as https://www.discourse.org/, so that the community will have a real place to give feedback. Thanks for your work on this product, and it's underlying purpose: our privacy!

**Comment by markcellus** [comment]
Yes. It is very hard to follow discussions on that closed-source product or know about when new topics are created. If the community isn't notified of new posts, the author loses out on upvotes.

**Comment by itsnotsaved** [comment]
Simple exact alternative to uservoice https://fider.io/
https://github.com/getfider/fider

**Comment by pYFZOS5** [comment]
Discourse - https://github.com/discourse/discourse (free & 100% opensource) and @julianfairfax said above

It has voting feature too. you can keep a separate category or subcategories[1] for feature requests (and for each apps).
You can use default-like feature or add [discourse voting plugin](https://meta.discourse.org/t/discourse-voting/40121?u=joebuhlig) or keep them both enabled [show both like and vote button on topics](https://meta.discourse.org/t/show-both-like-and-vote-button-on-topics/101252)

[1][How to create a Sub-Category](https://meta.discourse.org/t/how-to-create-a-sub-category/71558)
![12f3017b91cdf8652813ac5a83ddd712c5053271](https://user-images.githubusercontent.com/73447325/183559668-483c23c7-7fb9-4b3a-a19d-374a43bdf62a.gif)
[2][How do I display the subcategories inside of the parent by default?](https://meta.discourse.org/t/how-do-i-display-the-subcategories-inside-of-the-parent-by-default/166514)
![fcf2e561d78a8c28285aac36d318a1ef3bd81a33_2_225x250](https://user-images.githubusercontent.com/73447325/183559339-ddfc7bdd-bec3-4919-a712-8a1583ee4721.png)

Also you can do following things in different categories,
1. Announcements to community
2. You can merge any existing forum parts, commenting walls to this
3. Beta program discussions and etc (you don't need reddit)
4. You can keep a separate developer portal / discussion (category)
5. You can keep a separate category for translation related discussions and if required, enable **live chat feature** too for that.
[Discourse Chat](https://meta.discourse.org/t/discourse-chat/230881) (you can keep it as invite-only or with restricted access)
(once again, you're using discord for that. same as uservoice) past community asked you to use mattermost (free & opensource), zulip (free and opensource), revoltchat (100% free & opensource) etc.
6. You can keep a separate discussion for proton tools & SDK (pmcrypto, gluon, openpgp, proton bridge, sdks)

![52239250-04ad8280-289c-11e9-9e42-574f6eaab9d7](https://user-images.githubusercontent.com/78912974/183561469-e12e97fd-696f-4451-8166-c43964aabb51.png)

One-for-All

Who use discourse? https://www.discourse.org/customers
Asana, Samsung, Atlassian, Western Digital, Zoom, Twitter, Unreal engine

**Note: We do not like current behavior.**

**Comment by pYFZOS5** [comment]
Proton feature requests - Uservoice
Beta discussion / User questions - Reddit
Proton i18n discussions - Discord
(3rd is the new thing for last year)

Ping @EpokK @twiss 
i can't find anyone else to ping here to take attention for that yrs ignoring thing.

**Comment by julianfairfax** [comment]
You could do what Standard Notes do and simply setup a GitHub repository to use as your forum through the issues or discussions function. See https://github.com/standardnotes/forum

**Comment by twiss** [comment]
Hi there, I've passed it on internally. I don't think anyone at Proton is a huge fan of the current solution, either :)

**Comment by RokeJulianLockhart** [comment]
> https://github.com/ProtonMail/WebClients/issues/260#issuecomment-1220811255

@twiss and @julianfairfax, in which case, why is this closed as unplanned? It seems strange to not publicly track this, when it's so important to the public specifically.

Every other organisation similar to Proton uses Discourse:

1. Tor (browser)
1. Signal (messenger)
1. Fedora (OS)
1. Mozilla (browser)
1. KDE (DE)
1. GNOME (DE)
1. Murena (AOSP distribution)
1. GRAMPS (genealogy)
1. NextCloud
1. OwnCloud
1. Blender
1. Flatpak
1. EndeavourOS
1. The Document Foundation
1. Rust

None of these utilize UserVoice.

**Comment by julianfairfax** [comment]
It was open for almost two years without progess. It is closed as unplanned because it obviously is. And this is the repo for the web clients

**Comment by RokeJulianLockhart** [comment]
> And this is the repo for the web clients

@julianfairfax, that's a good point. Does a Proton UserVoice request to move to another platform exist? I've been unable to locate one thus far, but its GUI is unintuitive.
